source/style-guide/style/callouts.txt

```diff
@@ -0,0 +1,99 @@
+.. _callouts:
+
+========================
+Callouts and Admonitions
+========================
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Callouts or admonitions emphasize important or
+helpful information. Use callouts sparingly to avoid clutter on
+the page and to preserve visual impact. 
+
+.. list-table::
+   :widths: 20 60 30
+   :header-rows: 1
+
+   * - Callout Type
+     - Description
+     - Directive
+   * - Tip
+     - Provides useful information that might improve product
+       performance or make procedures easier to follow. Tips provide
+       the following benefits:
+
+       - Help users learn techniques or procedures
+       - Show alternative ways of doing something
+       - Provide shortcuts
+       - Provide helpful (but not essential) information
+
+     - ``.. tip:: <title>``
+
+   * - Note
+     - Provides information that emphasizes or supplements information
+       in the text. A note can provide information that applies only in
+       certain cases.
+     - ``.. note:: <title>``
+   * - Important
+     - Presents an important or essential point. As a rule, users must
+       pay attention to important callouts to complete a task or
+       understand a topic.
+     - ``.. important:: <title>``
+   * - Warning
+     - Alerts users to potential hazards or highlights critical
+       information. Use a warning for situations in which users could
+       lose data, compromise data integrity, or disrupt operations if
+       they don't follow instructions carefully.
+     - ``.. warning:: <title>``
+   * - Example
+     - Presents a scenario that illustrates or demonstrates a concept in
+       the text.
+     - ``.. example:: <title>``
+   * - See
+     - Provides a link to a topic or section directly related to the
+       preceding text. You might use ``see`` to link to content with
+       necessary procedures, definitions, or deployment considerations
+       and impacts.
+     - ``.. see::``
+   * - See also
+     - Provides a link to topics or sections that are generally related 
+       to or provide useful context for the preceding text. You might
+       use ``seealso`` to link to content that provides background
+       information, deeper context, or common related tasks.
+     - ``.. seealso::``
+
+
+When creating callouts, use the following guidelines:
+
+-  Use the callout type to convey the kind of information you want 
+   to communicate. Don't rely on color to convey intensity.
+
+-  Use the correct {+rst+} directive to create the callout. Don't 
+   attempt to create callouts using boldface, headings, or custom
+   :abbr:`CSS`.
```

Comment:
**suggestion:** If you use the `:abbr:` role, you have to provide what it is abbreviating:

```suggestion
   :abbr:`CSS (Cascading Style Sheets)`.
```